assignments/prerequisites/python-programming-advanced.ipynb

```diff
@@ -54,7 +54,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Merge this together  with above cell. And always give an empty line before the assert.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>